packages/codemods/src/cli.ts

```diff
@@ -0,0 +1,55 @@
+import { $ } from 'execa';
+import sade from 'sade';
+import { fs, path } from 'zx';
+import { _dirname, errorMessage, logMessage } from './helpers';
+
+const packageJson = fs.readJsonSync(path.resolve(_dirname, './package.json'));
+
+export default async function cli(args: string[]) {
+  sade('spirit-codemods', true)
+    .version(packageJson.version)
+    .describe(packageJson.description)
+    .option('-p, --path', 'Path to the code to be transformed')
+    .example('-p ./')
+    .option('-t, --transformation', 'Codemod transformation name to run')
+    .example('-c v2/web-react/codemodName')
+    .option('-e, --extensions', 'Extensions to look for when transforming files, default: ts,tsx,js,jsx')
+    .example('-e ts, tsx, js, jsx')
+    .option('--ignore', 'Ignore files or directories, default: **/node_modules/**')
```

Comment:
```suggestion
    .option('-i, --ignore', 'Ignore files or directories, default: **/node_modules/**')
```

You have `-i` in the example below.

Reply:
added, thank you